packages/backend/src/tokens/interstitialRule.ts

```diff
@@ -87,7 +88,17 @@ export const isNormalSignedOutState: InterstitialRule = async options => {
 
 // This happens when a signed in user visits a new subdomain for the first time. The uat will be available because it's set on naked domain, but session will be missing. It can also happen if the cookieToken is manually removed during development.
 export const hasClientUatButCookieIsMissingInProd: InterstitialRule = async options => {
-  const { clientUat, cookieToken } = options as any;
+  const {
+    clientUat,
+    isSatellite,
+    cookieToken,
+    isSynced = false,
```

Comment:
@colinclerk  In case of remix, it is there already yes. I just added that for safety. Because in the types I defined it as optional. How about adding `!!isSynced` ?
```suggestion
  const {
    clientUat,
    isSatellite,
    cookieToken,
    isSynced
  } = options as AuthenticateRequestOptionsWithExperimental;

  if (clientUat && Number.parseInt(clientUat) > 0 && !cookieToken && !!isSatellite && isSynced) {
    return interstitial(options, AuthErrorReason.SatelliteCookieMissing);
  }
```

Reply:
`isSynced` needs to be defined to a boolean before the rule runs